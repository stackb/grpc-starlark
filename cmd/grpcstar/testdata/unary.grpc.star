pb = proto.package("example.routeguide.RouteGuide")
print("pb:", pb)
