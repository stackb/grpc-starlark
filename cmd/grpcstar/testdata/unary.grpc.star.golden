pb: <proto.Package "example.routeguide.RouteGuide">
